controller.go

```diff
@@ -106,14 +119,16 @@ type driverData struct {
 type driverTable map[string]*driverData
 type networkTable map[types.UUID]*network
 type endpointTable map[types.UUID]*endpoint
-type sandboxTable map[string]*sandboxData
+type containerTable map[string]*sandboxData
```

Comment:
am generally confused with the naming convention between container and sandbox.
Is there any specific logic behind these names that I should look for ?

For example, why is sandboxData map named `containerTable` ?